test/runtests.jl

```diff
@@ -25,17 +36,20 @@ using ConicBenchmarkUtilities
     x_sol = MathProgBase.getsolution(m)
     objval = MathProgBase.getobjval(m)
 
+    #=
+    # drop Convex.jl
```

Comment:
Better to just delete the code instead of leaving a commented block that's unlikely to be revived. The code is stored in the git history if we need it.